lib/sequent/test/command_handler_helpers.rb

```diff
@@ -114,14 +119,24 @@ def given_events *events
         @event_store.given_events(events)
       end
 
+
       def when_command command
         raise "@command_handler is mandatory when using the #{self.class}" unless @command_handler
         raise "Command handler #{@command_handler} cannot handle command #{command}, please configure the command type (forgot an include in the command class?)" unless @command_handler.handles_message?(command)
-        @command_handler.handle_message(command)
+        begin
+          @command_handler.handle_message(command)
+        rescue StandardError => e
```

Comment:
`rescue => e` only rescues `StandardError` derived errors; no need to specify that here.